app.js

```diff
@@ -0,0 +1,60 @@
+var projectData = [
+  {
```

Comment:
As a heads up, next week we will be transferring our data into a separate json file.